jobs/business-sync/src/business_sync/config.py

```diff
@@ -63,20 +63,24 @@ class _Config(object):  # pylint: disable=too-few-public-methods
             f"postgresql+pg8000://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
         )
 
-    WAREHOUSE_USER = os.getenv("WAREHOUSE_USER", "")
-    WAREHOUSE_PASSWORD = os.getenv("WAREHOUSE_PASSWORD", "")
-    WAREHOUSE_HOST = os.getenv("WAREHOUSE_HOST", "")
-    WAREHOUSE_PORT = int(os.getenv("WAREHOUSE_PORT", "5432"))  # POSTGRESQL
-    WAREHOUSE_DB_NAME = os.getenv("WAREHOUSE_DB_NAME", "")
-    if WAREHOUSE_DB_UNIX_SOCKET := os.getenv("WAREHOUSE_DB_UNIX_SOCKET", None):
-        WAREHOUSE_URI = f"postgresql+pg8000://{WAREHOUSE_USER}:{WAREHOUSE_PASSWORD}@/{WAREHOUSE_DB_NAME}?unix_sock={WAREHOUSE_DB_UNIX_SOCKET}/.s.PGSQL.5432"
-    else:
-        WAREHOUSE_URI = f"postgresql+pg8000://{WAREHOUSE_USER}:{WAREHOUSE_PASSWORD}@{WAREHOUSE_HOST}:{WAREHOUSE_PORT}/{WAREHOUSE_DB_NAME}"
+    WAREHOUSE_DB_UNIX_SOCKET = os.getenv("WAREHOUSE_DB_UNIX_SOCKET")
+    WAREHOUSE_CREDENTIALS_FILE = os.getenv("WAREHOUSE_CREDENTIALS_FILE")
+    AUTH_PROXY_CONNECT = os.getenv("AUTH_PROXY_CONNECT")
```

Comment:
you are referencing undefined variable here from jobs/business-sync/devops/vaults.gcp.env